docs/promscale_alerting.md

```diff
@@ -0,0 +1,226 @@
+# Promscale Alerting
+
+This file contains alerting rules which are symptom based. We recommend monitoring your Promscale deployments
+and include the following rules as part of your alerting environment.
+
+## Rules
+
+Copy the below in a `promscale_alerts.yaml` file.
+
+```yaml
+groups:
+- name: Promscale
+  rules:
+  - alert: PromscaleDown
+    expr: absent(up{job=~".*promscale.*"})
+    for: 2m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: Promscale is down
+      description: No Promscale instance was found.
+# Ingest
+  - alert: PromscaleIngestHighErrorRate
+    expr: |
+      (
+        sum by (job, instance, type) (
+          rate(promscale_ingest_requests_total{code=~"5.."}[5m])
+        )
+      /
+        sum by (job, instance, type) (
+          rate(promscale_ingest_requests_total[5m])
+        )
+      ) > 0.05
+    for: 30s # Assuming one scrape interval. 
```

Comment:
This wasn't in the DD and shouldn't be put here. There is no reason to wait for this alert.